graphgenerator/utils/tweet_extraction.py

```diff
@@ -3,15 +3,15 @@
 import json
 
 
-def compute_bot_score_from_user_info(user_info, compute_botscore=True):
+def compute_bot_score_from_user_info(user_info, compute_botscore):
     """
     Compute botscore using botfinder using user information
     the function findbot_rawjson from the package is used rather than the CLI
     """
     if compute_botscore:
         return json.loads(findbot_rawjson(json.dumps(user_info)))["botScore"]
     else:
-        return float("nan")
+        return 99999
```

Comment:
I would rather return undefined, or not include botScore at all in result when parameter is not passed